test/addons/rook-external-cluster/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+resources:
+  - ${base_url}/cluster-external.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+patches:
+  - target:
+      kind: CephCluster
+      name: rook-ceph-external
+      namespace: rook-ceph-external
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: my-cluster
```

Comment:
Can you add a comment for every patch, explaining why we keep this patch?

For this patch I guess the reason is to use the same cluster name so we can reuse the existing code to create the pool.